src/config.rs

```diff
@@ -679,7 +671,6 @@ impl Default for CompactorOptions {
             compaction_scheduler: default_compaction_scheduler(),
             max_concurrent_compactions: 4,
             compaction_runtime: None,
-            clock: default_clock(),
```

Comment:
you can probably remove the `default_clock` function definition, I don't think it's used anywhere anymore after this change.

Reply:
It's still used in the default implementation of DbConfig (also I think clippy checks for unused functions and fails the bulks of there are any).